pkg/kubelet/util/manager/watch_based_manager.go

```diff
@@ -121,6 +121,15 @@ func (i *objectCacheItem) startReflector() {
 	i.waitGroup.Wait()
 	i.waitGroup.Add(1)
 	defer i.waitGroup.Done()
+	resourceVersion := ""
+	// store has only a single item at most
```

Comment:
If so, why do we loop over it?

Reply:
`objectCacheItem` has no information about key, we can't get by key from store.